ci/unit_tests.sh

```diff
@@ -20,25 +20,8 @@ fi
 SELECTED_TEST_SUITE=$1
 
 if [[ $SELECTED_TEST_SUITE == $"java" ]]; then
-  SONAR_ARGS=()
-
-  if [[ $(echo $ENABLE_SONARQUBE | tr '[:lower:]' '[:upper:]') == "TRUE" ]]; then
-    SONAR_ARGS=(
-      "jacocoTestReport"
-      "sonar"
-      "-Dsonar.token=${SONAR_TOKEN}"
-      "-Dsonar.host.url=https://sonar.elastic.dev"
-      "-Dsonar.projectKey=elastic_logstash_AYm_nEbQaV3I-igkX1q9"
-      "-Dsonar.projectName=logstash"
-      "-Dsonar.pullrequest.key=$PULL_ID"
-      "-Dsonar.pullrequest.branch=$SOURCE_BRANCH"
-      "-Dsonar.pullrequest.base=$TARGET_BRANCH"
-      "-Dsonar.scm.revision=$COMMIT_SHA"
-    )
-  fi
-
   echo "Running Java Tests"
-  ./gradlew javaTests "${SONAR_ARGS[@]}" --console=plain --warning-mode all
+  ./gradlew javaTests jacocoTestReport --console=plain --warning-mode all
```

Comment:
By hardcoding the `jacocoTestReport` and removing the `$ENABLE_SONARQUBE` conditional, how can we skip running it when `ci/unit_tests.sh` are triggered in the `aarch64_pipeline.yml` or the `exhaustive_tests` pipeline mentioned in the comment above.

Reply:
`jacocoTestReport` just creates a file with coverage data without sending it to sonarqube. Should we remove it for other types of tests?